runtime/oti/vmconstantpool.xml

```diff
@@ -209,13 +210,16 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<fieldref class="java/lang/StackWalker$StackFrameImpl" name="frameModule" signature="Ljava/lang/Module;" versions="9-"/>
 
 	<fieldref class="java/lang/Thread" name="priority" signature="I"/>
-	<fieldref class="java/lang/Thread" name="isDaemon" signature="Z"/>
-	<fieldref class="java/lang/Thread" name="runnable" signature="Ljava/lang/Runnable;"/>
+	<fieldref class="java/lang/Thread" name="daemon" signature="Z" flags="opt_ojdkThreadSupport"/>
+	<fieldref class="java/lang/Thread" name="isDaemon" signature="Z" flags="!opt_ojdkThreadSupport"/>
+	<fieldref class="java/lang/Thread" name="target" signature="Ljava/lang/Runnable;" flags="opt_ojdkThreadSupport"/>
+	<fieldref class="java/lang/Thread" name="runnable" signature="Ljava/lang/Runnable;" flags="!opt_ojdkThreadSupport"/>
```

Comment:
Is this used? Maybe runnable/target doesn't need to be in the constant pool if they aren't used.

Reply:
Removed.